src/terminalExecutor.ts

```diff
@@ -2,72 +2,37 @@
 
 import * as vscode from 'vscode';
```

Comment:
Maybe TerminalExecutor should become a part of TerminalRunner? it just has static functions which are used only from TerminalRunner

Reply:
it's separate as its functionality is quite independent. that's why separate it out